docs/DeveloperGuide.md

```diff
@@ -350,6 +350,16 @@ Use case ends.
 
 Use case ends.
 
+**Use case: UC04 - Find a Buyer, Supplier or Deliverer**
```

Comment:
Good job to update the developer guide concurrently with the changes in the code base.